code/game/objects/items/twohanded.dm

```diff
@@ -1017,3 +1017,256 @@
 		C.change_view(CONFIG_GET(string/default_view))
 		user.client.pixel_x = 0
 		user.client.pixel_y = 0
+
+/obj/item/twohanded/electrostaff
+	icon = 'icons/obj/estaff.dmi'
+	icon_state = "electrostaff_3"
+	item_state = "electrostaff_3"
+	lefthand_file = 'icons/mob/inhands/weapons/estaff_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/estaff_righthand.dmi'
+	name = "riot suppression electrostaff"
+	desc = "A large quarterstaff, with massive silver electrodes mounted at the end."
+	force = 10
+	damtype = BRUTE
+	w_class = WEIGHT_CLASS_GIGANTIC
+	slot_flags = ITEM_SLOT_BACK
+	sharpness = FALSE
+	force_unwielded = 5
+	force_wielded = 10
+	throwforce = 1
+	throw_speed = 1
+	block_chance = 50
+	materials = list(MAT_METAL=1000)
```

Comment:
>With silver, yet has no silver

Reply:
don't add silver to the constituent materials, it allows for silver at roundstart by deconning them

Reply:
@lolman360 surgery tables have a bunch btw